src/main/java/org/apache/commons/text/names/Name.java

```diff
@@ -26,14 +26,16 @@
 public final class Name {
 
     private final String leadingInitial;
+    private final String salutation;
     private final String firstName;
     private final String nickName;
     private final String middleName;
     private final String lastName;
     private final String suffix;
 
-    Name(String leadingInitial, String firstName, String nickName, String middleName, String lastName, String suffix) {
+    public Name(String leadingInitial, String salutation, String firstName, String nickName, String middleName, String lastName, String suffix) {
```

Comment:
yup, same here. I think I did this for unit tests. not too sure though. I'll revert and we can discuss later.